api/db/seeds/data/users-builder.js

```diff
@@ -13,7 +13,7 @@ function usersBuilder({ databaseBuilder }) {
   });
 
   databaseBuilder.factory.buildUser.withRawPassword({
-    id: PIX_MASTER_ID,
+    id: PIX_SUPER_ADMIN_ID,
     firstName: 'Pix',
     lastName: 'Master',
     email: 'pixmaster@example.net',
```

Comment:
Autant aller jusqu'au bout ! 🔥
```suggestion
    firstName: 'Super',
    lastName: 'Admin',
    email: 'superadmin@example.net',
```

Reply:
hmm comme beaucoup de gens utilisent cet utilisateur on s'était dit que c'était plus simple de laisser ces info là 🤔 ... ou alors faut qu'on communique ces changements à toutes les équipes